test/transferSpec.js

```diff
@@ -120,15 +117,14 @@ describe('Plugin transfers (optimistic)', function () {
       this.pluginA.send(Object.assign({
         id: id,
         amount: '1.0',
-        data: new Buffer(''),
-        noteToSelf: new Buffer(''),
+        data: '',
+        noteToSelf: '',
         executionCondition: undefined,
-        cancellationCondition: undefined,
         expiresAt: undefined
       }, transferA)).catch(done)
     })
 
-    it('should reject optimistic transfer with repeat id', function (done) {
+    it('should not reject optimistic transfer with repeat id', function (done) {
```

Comment:
Right, note that a repeat id with different transfer details is still an error. Maybe this behavior is going to be too complicated for plugin authors, but it's the most correct / safest I think. So let's see how it plays out.